### ISSUE TITLE
Cannot reproduce the recognition accuracy of certain expressions in the paper

### ISSUE THREAD

**Issue by AMG024** [amirhfarzaneh/dacl#8]
I used the msceleb weights pre-training model parameters in the paper to train the model’s accuracy on a certain expression that is not as accurate as the recognition accuracy in the confusion matrix in the paper. The parameters used are the default parameters in the paper, so I want to know how In order to reproduce the recognition accuracy of each expression in the confusion matrix of the paper

**Comment by nothingeasy** [comment]
I meet the same problem . Does anyone reproduce the result in the paper?

**Comment by anony-mous-rep** [comment]
I also meet the same problem . Is my data set error? I use RAF-DB data set to train, only get the accuracy of 86.44%.

**Comment by nothingeasy** [comment]
Anyone try this code？  How about the resulut？ Please help~

**Comment by yun-kai** [comment]
I use RAF-DB data set to train and get the accuracy of 86.25%, and use AffectNet data set to train and get the accuracy of 56.8%.
I don't know why the accuracy of AffectNet is too far from the paper result. Does anyone meet same problem?

**Comment by hhwwxx11** [comment]
现在这个ms-celeb权重预训练模型无法下载，可以分享一下吗

**Comment by hhwwxx11** [comment]
Now the ms-cleb weight pre-training model cannot be downloaded. Can anyone share it?

**Comment by hhwwxx11** [comment]
Now the ms cleb weight pre training model cannot be downloaded, can you share it?

**Comment by hhwwxx11** [comment]
Now the ms-cleb weight pre-training model cannot be downloaded, can you share it?

**Comment by hhwwxx11** [comment]
Now the ms-celeb weight pre-training model cannot be downloaded, can you share it?

**Comment by hhwwxx11** [comment]
> 我使用论文中的msceleb权重预训练模型参数来训练模型在某个表达式上的准确率，该表达式的准确率不如论文中混淆矩阵中的识别准确率。使用的参数是论文中的默认参数，所以想知道如何才能重现论文的混淆矩阵中每个表达式的识别准确率
Hello, the link of the ms-celeb pre-training model in the paper has failed. Can you send me this pre training model?

**Comment by lbbbbbbbbbb** [comment]
> 

请问你有了么，链接还是失效

**Comment by ZechengLi19** [comment]
> Now the ms-celeb weight pre-training model cannot be downloaded, can you share it?

Same question！

**Comment by ZechengLi19** [comment]
> > 
> 
> 请问你有了么，链接还是失效

you can check this link https://github.com/zyh-uaiaaaa/relative-uncertainty-learning

**Comment by amirhfarzaneh** [comment]
link fixed. thanks for your patience!
